logos/src/logos/dbutils/dbmanager.py

```
@@ -3467,6 +3472,165 @@ def create_user(self, username: str, prename: str, name: str, email: str, role:
+                        continue
+                    self.add_team_member(team_obj["id"], user_dict["id"])
+                    summary["created"] += 1
+                    rows_out.append({
+                        "email": email,
+                        "username": user_dict["username"],
+                        "apiKey": logos_key,
+                        "team": team,
+                        "status": "created",
+                        "error": None,
+                    })
+            except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)